diff -urN dpkg-1.6.15.orig/utils/start-stop-daemon.c dpkg-1.6.15/utils/start-stop-daemon.c
--- dpkg-1.6.15.orig/utils/start-stop-daemon.c	2000-01-09 02:31:18.000000000 +0100
+++ dpkg-1.6.15/utils/start-stop-daemon.c	2003-08-30 22:38:37.000000000 +0200
@@ -161,34 +161,34 @@
 static void
 do_help(void)
 {
-	printf("\
-start-stop-daemon for Debian GNU/Linux - small and fast C version written by\n\
-Marek Michalkiewicz <marekm@i17linuxb.ists.pwr.wroc.pl>, public domain.\n"
-VERSION "\n\
-\n\
-Usage:
-  start-stop-daemon -S|--start options ... -- arguments ...\n\
-  start-stop-daemon -K|--stop options ...\n\
-  start-stop-daemon -H|--help\n\
-  start-stop-daemon -V|--version\n\
-\n\
-Options (at least one of --exec|--pidfile|--user is required):
-  -x|--exec <executable>        program to start/check if it is running\n\
-  -p|--pidfile <pid-file>       pid file to check\n\
-  -c|--chuid <name|uid[:group|gid]>
-  		change to this user/group before starting process\n\
-  -u|--user <username>|<uid>    stop processes owned by this user\n\
-  -n|--name <process-name>      stop processes with this name\n\
-  -s|--signal <signal>          signal to send (default TERM)\n\
-  -a|--startas <pathname>       program to start (default is <executable>)\n\
-  -b|--background               force the process to detach\n\
-  -m|--make-pidfile             create the pidfile before starting\n\
-  -t|--test                     test mode, don't do anything\n\
-  -o|--oknodo                   exit status 0 (not 1) if nothing done\n\
-  -q|--quiet                    be more quiet\n\
-  -v|--verbose                  be more verbose\n\
-\n\
-Exit status:  0 = done  1 = nothing done (=> 0 if --oknodo)  2 = trouble\n");
+	printf("\n"
+"start-stop-daemon for Debian GNU/Linux - small and fast C version written by\n"
+"Marek Michalkiewicz <marekm@i17linuxb.ists.pwr.wroc.pl>, public domain.\n"
+VERSION "\n"
+"\n"
+"Usage:\n"
+"  start-stop-daemon -S|--start options ... -- arguments ...\n"
+"  start-stop-daemon -K|--stop options ...\n"
+"  start-stop-daemon -H|--help\n"
+"  start-stop-daemon -V|--version\n"
+"\n"
+"Options (at least one of --exec|--pidfile|--user is required):\n"
+"  -x|--exec <executable>        program to start/check if it is running\n"
+"  -p|--pidfile <pid-file>       pid file to check\n"
+"  -c|--chuid <name|uid[:group|gid]>\n"
+"  		change to this user/group before starting process\n"
+"  -u|--user <username>|<uid>    stop processes owned by this user\n"
+"  -n|--name <process-name>      stop processes with this name\n"
+"  -s|--signal <signal>          signal to send (default TERM)\n"
+"  -a|--startas <pathname>       program to start (default is <executable>)\n"
+"  -b|--background               force the process to detach\n"
+"  -m|--make-pidfile             create the pidfile before starting\n"
+"  -t|--test                     test mode, don't do anything\n"
+"  -o|--oknodo                   exit status 0 (not 1) if nothing done\n"
+"  -q|--quiet                    be more quiet\n"
+"  -v|--verbose                  be more verbose\n"
+"\n"
+"Exit status:  0 = done  1 = nothing done (=> 0 if --oknodo)  2 = trouble\n");
 }
 
 
