--- dpkg-1.6.14/doc/Makefile.in~	Tue May 16 17:24:23 2000
+++ dpkg-1.6.14/doc/Makefile.in	Mon Jul 31 19:02:11 2000
@@ -8,7 +8,7 @@
 
 MAN5PAGES	= deb-control.5 deb-old.5 deb.5
 
-all: internals.html all-recursive
+all: all-recursive
 
 clean: clean-recursive
 	$(RM) $(srcdir)/manuals-version
--- dpkg-1.6.14/doc/Makefile.in~	Mon Jul 31 19:05:04 2000
+++ dpkg-1.6.14/doc/Makefile.in	Mon Jul 31 19:07:11 2000
@@ -33,11 +33,6 @@
 		if test -f $$i ; then d= ; else d="$(srcdir)/" ; fi ; \
 		$(INSTALL_DATA) $$d$$i $(DESTDIR)$(man5dir) ; \
 	done
-	$(mkinstalldirs) $(DESTDIR)$(docdir)/internals/
-	set -e ; for i in internals.html/* ; do \
-		bn=`basename $$i` ; \
-		$(INSTALL_DATA) $$i $(DESTDIR)$(docdir)/internals/$$bn ; \
-	done
 	$(INSTALL_DATA) $(srcdir)/ChangeLog $(DESTDIR)$(docdir)/ChangeLog.manuals
 	set -e ; for i in ChangeLog THANKS TODO ; do \
 		$(INSTALL_DATA) $(top_srcdir)/$$i $(DESTDIR)$(docdir) ; \
