--- scripts/Makefile.in.old	Sun Nov  4 17:54:04 2001
+++ scripts/Makefile.in	Sun Nov  4 17:54:09 2001
@@ -84,7 +84,7 @@
 		$(INSTALL_DATA) $$d$$i $(DESTDIR)/$(man1dir) ; \
 	done
 	set -e ; for i in $(MAN_SOURCE_ALIASES) ; do \
-		echo ".so man1/dpkg-source.1" > $(DESTDIR)$(man1dir)/$$i ; \
+		echo ".so dpkg-source.1" > $(DESTDIR)$(man1dir)/$$i ; \
 		chmod 644 $(DESTDIR)$(man1dir)/$$i ; \
 	done
 	$(mkinstalldirs) $(DESTDIR)/$(man8dir)
