--- dpkg-1.6.15/configure.in.orig	Fri Oct 19 22:48:16 2001
+++ dpkg-1.6.15/configure.in	Fri Oct 19 22:50:56 2001
@@ -49,8 +49,8 @@
 fi
 AC_DEFINE_UNQUOTED(PACKAGE, "$PACKAGE", [Name of package])
 AC_DEFINE_UNQUOTED(VERSION, "$VERSION", [Version number of package])
-AC_REQUIRE([AC_ARG_PROGRAM])
-AC_REQUIRE([AC_PROG_MAKE_SET])
+AC_ARG_PROGRAM
+AC_PROG_MAKE_SET
 
 AC_MSG_CHECKING(dpkg version)
 AC_MSG_RESULT($VERSION)
diff -Nur dpkg-1.6.15.orig/acinclude.m4 dpkg-1.6.15/acinclude.m4
--- dpkg-1.6.15.orig/acinclude.m4	Thu Jan  1 01:00:00 1970
+++ dpkg-1.6.15/acinclude.m4	Fri Oct 19 22:54:18 2001
@@ -0,0 +1,91 @@
+dnl Bunch of extra macros to make dpkg more portable
+dnl Copyright (C) 1999 Wichert Akkerman <wakkerma@debian.org>
+dnl
+dnl This program is free software; you can redistribute it and/or modify
+dnl it under the terms of the GNU General Public License as published by
+dnl the Free Software Foundation; either version 2, or (at your option)
+dnl any later version.
+dnl
+dnl This program is distributed in the hope that it will be useful,
+dnl but WITHOUT ANY WARRANTY; without even the implied warranty of
+dnl MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+dnl GNU General Public License for more details.
+dnl
+dnl You should have received a copy of the GNU General Public License
+dnl along with this program; if not, write to the Free Software
+dnl Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA
+dnl 02111-1307, USA.
+
+AC_DEFUN(AC_HAVE_SYSINFO,
+[AC_CACHE_CHECK(whether sysinfo is available,
+ ac_cv_func_sysinfo,
+[ AC_CHECK_FUNC(sysinfo,ac_cv_func_sysinfo=yes,ac_cv_func_sysinfo=no)])
+if test "$ac_cv_func_sysinfo" = "yes" ; then
+  AC_DEFINE(HAVE_SYSINFO)
+fi
+])
+
+AC_DEFUN(AC_MEMINFO_IN_SYSINFO,
+[AC_REQUIRE([AC_HAVE_SYSINFO])dnl
+ if test "$ac_cv_header_sys_sysinfo" = "" ; then
+   AC_CHECK_HEADERS(sys/sysinfo.h)
+ fi
+ AC_CACHE_CHECK(whether struct sysinfo contains memory information,
+ ac_cv_meminfo_in_sysinfo,
+[AC_TRY_COMPILE([
+#ifdef HAVE_SYS_SYSINFO_H
+#include <sys/sysinfo.h>
+#endif
+], [struct sysinfo si ; si.freeram;si.sharedram;si.bufferram;],
+  ac_cv_meminfo_in_sysinfo=yes, ac_cv_meminfo_in_sysinfo=no)])
+ if test "$ac_cv_have_sysinfo_meminfo" = yes ; then
+   AC_DEFINE(MEMINFO_IN_SYSINFO)
+ fi
+])
+
+
+
+dnl Moved from configure.in, modified to use AC_DEFUN
+dnl -- Tom Lees <tom@lpsg.demon.co.uk>
+
+dnl DPKG_CACHED_TRY_COMPILE(<description>,<cachevar>,<include>,<program>,<ifyes>,<ifno>)
+AC_DEFUN(DPKG_CACHED_TRY_COMPILE,[
+ AC_MSG_CHECKING($1)
+ AC_CACHE_VAL($2,[
+  AC_TRY_COMPILE([$3],[$4],[$2=yes],[$2=no])
+ ])
+ if test "x$$2" = xyes; then
+  true
+  $5
+ else
+  true
+  $6
+ fi
+])
+
+dnl DPKG_C_GCC_TRY_WARNS(<warnings>,<cachevar>)
+AC_DEFUN(DPKG_C_GCC_TRY_WARNS,[
+ AC_MSG_CHECKING([GCC warning flag(s) $1])
+ if test "${GCC-no}" = yes
+ then
+  AC_CACHE_VAL($2,[
+   oldcflags="${CFLAGS-}"
+   CFLAGS="${CFLAGS-} ${CWARNS} $1 -Werror"
+   AC_TRY_COMPILE([
+#include <string.h>
+#include <stdio.h>
+],[
+    strcmp("a","b"); fprintf(stdout,"test ok\n");
+], [$2=yes], [$2=no])
+   CFLAGS="${oldcflags}"])
+  if test "x$$2" = xyes; then
+   CWARNS="${CWARNS} $1"
+   AC_MSG_RESULT(ok)
+  else
+   $2=''
+   AC_MSG_RESULT(no)
+  fi
+ else
+  AC_MSG_RESULT(no, not using GCC)
+ fi
+])
